cumulusci/cli/options.py

```diff
@@ -0,0 +1,28 @@
+"""Process environment variables and global configurations for some options.
+
+Returns option values in the following order of precedence:
+1. Global configuration options
+2. Click Option
+3. Environment variables
+4. Click Option default value
+"""
+
+
+from cumulusci.core.config import BaseGlobalConfig
+
+# CLI Environment Variables
+CCI_ENV_PREFIX = "CCI__"
+CLI_ENV_PREFIX = CCI_ENV_PREFIX + "CLI__"
+NO_PROMPT_ENV = CLI_ENV_PREFIX + "ALWAYS_RECREATE"
```

Comment:
Can we:
1. Make this more specific, i.e. "CCI__CLI__RECREATE_EXPIRED_SCRATCH_ORGS"
2. Update the name and help for the `--no-prompt` options to match? I'm okay that this is a backwards-incompatible change; I doubt the `--no-prompt` option is used much and I consider it a bug that the option does something different than what its name and help suggest.